### DISCUSSION TITLE
Disable page loading effect

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2419] [Ideas]
Go to B**ackend > Catalog > Manage Categories**. Click a category name in the tree on the left side. For a fraction of a second a light gray background overlaps over the content of the page and in the middle of the screen a small window appears in which "Please Wait" message is written with an animated icon above. It's the well-known AJAX effect we're used to.

When Magento was designed before 2010, the servers were not performing like today. then loading a page took a long time and there were criticisms related to the speed of the platform. Such a visual effect could last for a good few seconds and came to calm the user. After 14 years things have changed, in my case when I quickly switch between categories the pages load instantly, but that effect is like a flash and it has become annoying. I was asked to disable it. Does anyone have a solution for this issue? 

I checked in today's last version cloned OM 1.9.4.x and really the effect has become disturbing, we should not use it anymore. We should be able to disable this effect in the config if we want it.

**Comment by justinbeaty** [comment]
It would be better to set a timeout on the effect. For example only show if the request is taking more than 100ms.

I will take a look at this tomorrow as I know somewhat where this is located in the JS files.

**Reply by addison74** [reply]
I agree. Disabling it completely would not be a good idea. It should appear only when the page loading time is greater than X ms. We could control this value in the config settings. On my test server with PHP 8.1 the pages load instantly. That effect is really annoying when you start to become aware of it.

**Comment by addison74** [comment]
Done in PR #2426  and #2475.
